examples/js/pmrem/PMREMGenerator.js

```diff
@@ -21,11 +19,13 @@ THREE.PMREMGenerator = ( function () {
 	scene.add( planeMesh );
 	scene.add( camera );
 
-	var PMREMGenerator = function ( sourceTexture, samplesPerLevel, resolution ) {
+	var PMREMGenerator = function ( sourceTexture, params = {} ) {
 
 		this.sourceTexture = sourceTexture;
-		this.resolution = ( resolution !== undefined ) ? resolution : 256; // NODE: 256 is currently hard coded in the glsl code for performance reasons
-		this.samplesPerLevel = ( samplesPerLevel !== undefined ) ? samplesPerLevel : 32;
+		this.sourceResolution = ( params.sourceResolution !== undefined ) ? params.sourceResolution : 512;
```

Comment:
Might be helpful to check for `resolution` for folks upgrading to this version if `sourceResolution` not found, e.g. `( params.sourceResolution !== undefined ) ? params.sourceResolution : params.resolution !== undefined ? params.resolution : 512;`

Reply:
good idea.